lib/widget/lottie/web/lottiestate.dart

```diff
@@ -92,7 +92,7 @@ class LottieState extends WidgetState<EnsembleLottie>
     bool repeat = widget.controller.repeat;
     bool autoPlay = widget.controller.autoPlay;
     if (widget.controller.width == null || widget.controller.height == null) {
-      throw LanguageError(
```

Comment:
you want to show this error so the AppDev knows to fix this. This is not a silent error